docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.7"
+services:
+  php-fpm:
+    volumes:
+      - spddata:/var/www/html/ 
+    build: 
+      context: . 
+      dockerfile: 'docker/php-fpm/Dockerfile'
+    networks:
+      - spdashboard
+    container_name: fpm
+  web:
+    build:
+      context: 'docker/httpd/'
+    depends_on:
+      - php-fpm
+    expose:
+      - "80"
+    networks:
+      - spdashboard
+    volumes:
+      - type: volume
```

Comment:
This could also use the short hand form above but should then become: `spddata:/var/www/html/:nocopy`